src/new_template_sushangagneja.html

```diff
@@ -0,0 +1,60 @@
+
+<body>
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.12/css/all.css" crossorigin="anonymous">
+<link href="https://fonts.googleapis.com/css?family=Cabin|Mansalva|Nunito&display=swap" rel="stylesheet">
+
```

Comment:
There are few unused fonts here.

Reply:
Cabin is being used for the designation. Mansalva for the name. Nunito for the phone number/email address.

Reply:
Ohh, my bad.
Sorry.